### ISSUE TITLE
ParamArray with variant array as a parameter , variant array has no elements in receiving method

### ISSUE THREAD

**Issue by FullValueRider** [WaynePhillipsEA/twinbasic#347]
**Describe the bug**
I have an instance of a variant containing an array being passed to a method that uses paramarray for its input. In this specific instance the variant containing an array reports that there are no elements when viewed in the receiving method in the debugger.

**To Reproduce**
Code is in the attached library.
[VBALib.zip](https://github.com/WaynePhillipsEA/twinbasic/files/6974073/VBALib.zip)

A check example in Module Pootle passes two arrays ( array(1,2,3,4,5), array(10,20,30,40,50)) to a method using paramarray for its input.  In the debugger, the arrays can be seen correctly in the recieving method, i.e. Paramarray(0) is the first array and Paramarray(1) is the second array.  The paramarray can be converted to a variant and further forwarded and the arrays are correctly forwarded again.  This is as expected.

In Module KVpTesting execution of ' Test01_ByRow__UseRow1AsKeysAndData' leads to myKvp.Add <array>, <enum> where the add method uses (ParamArray ipArgs() as variant) for input

When using the debugger to view 'Test01_ByRow__UseRow1AsKeysAndData'  the <array> is populated correctly but when received in the Add method (Kvp Class Method Add) ipArgs(0) , which should be the populated <array>, is identified as {dynamic array, zero elements}.

This may be yet another case of familiarity blindness but I'm posting here in case its a real bug.

**Comment by WaynePhillipsEA** [comment]
Yes, that's a bug involving fixed arrays.  Thanks for reporting.

For now, you can change 
`Dim myArray(1 To 6, 1 To 5) As Long`

to:
```
Dim myArray() As Long
ReDim myArray(1 To 6, 1 To 5)
```

Will be fixed by tomorrow.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.4198, thanks!
